operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md

```diff
@@ -0,0 +1,173 @@
+- [Boot set `rootfs_provider` parameters](#boot-set-rootfs_provider-parameters)
+  - [`<transport>`](#transport)
+  - [`<schema version>`](#schema-version)
+  - [`<IQN Domain>`](#iqn-domain)
+  - [`<DNS SRV record reference>`](#dns-service-srv-record-reference)
+  - [`<client discovery timeout in seconds>`](#client-discovery-timeout-in-seconds)
+  - [Example `rootfs_provider_passthrough`](#example-rootfs_provider_passthrough)
+- [`root=` kernel parameter](#root-kernel-parameter)
+- [Example session template input file](#example-session-template-input-file)
+- [Creating a BOS session using the new template](#creating-a-bos-session-using-the-new-template)
+
+## Boot set S3 parameters
+
+The session template boot set contains several [Simple Storage Service (S3)](../../glossary.md#simple-storage-service-s3) parameters.
+These are listed below, along with the appropriate values to use.
+
+- `type`: Set to `s3`
+- `path`: Set to `s3://<BUCKET_NAME>/<KEY_NAME>`
+  - `<BUCKET_NAME>` is set to `boot-images`
+  - `<KEY_NAME>` is set to the image ID that the [Image Management Service (IMS)](../../glossary.md#image-management-service-ims) created when it generated the boot artifacts.
+- `etag`: Set to `<etag\>`
+
+## Boot set `rootfs_provider` parameters
+
+The Scalable Boot Projection Service (SBPS) is an optional provider for the `rootfs` on compute nodes.
+
+For SBPS, the `rootfs_provider_passthrough` boot set parameter is customized according to the following format:
+
+```text
+rootfs_provider_passthrough=<transport>:<schema version>:<IQN Domain>:<DNS SRV record reference>:<client discovery timeout in seconds>
+```
+
+The following values need to be set in the boot set of the session template in order to make SBPS the `rootfs` provider:
+
+- `"rootfs_provider":` Set to `"sbps"`
+- To use the Node Management Network (NMN) for content projection,
+  - `"rootfs_provide_passthrough"`: Set to `"sbps:v1:iqn.2023-06.csm.iscsi:_sbps-nmn._tcp.local:300"`
+- To use the High Speed Network (HSN) for content projection,
+  - `"rootfs_provide_passthrough"`: Set to `"sbps:v1:iqn.2023-06.csm.iscsi:_sbps-hsn._tcp.local:300"`
+
+Note that `iqn.2023-06.csm.iscsi` is an example of the IQN domain. Provide an IQN domain specific to the customer.
+
+Note that `_sbps-hsn._tcp.local` can be used if it corresponds to the DNS SRV record. Otherwise, use the DNS SRV record.
+
+The variables used in this parameter represent the following:
+
+### `<transport>`
+
+File system network transport. For example, `sbps` or `dvs`.
+
+Set to `sbps` to use SBPS. Can be left as an empty string to use the default value `dvs`.
+
+### `<schema version>`
+
+The version of the SBPS schema. `v1` is the only supported version.
+
+### `<IQN Domain>`
+
+The domain name portion of the iSCSI Qualified Name (IQN).
+
+For example, an IQN might look like this:
+
+```text
+iqn.2001-04.com.example:storage:diskarrays-sn-a8675309
+```
+
+In this example:
+
+* `iqn.2001-04.com.example` is the IQN domain.
```

Comment:
Might help to make match the IQN examples for the targets they'll see on a CSM system, based on worker name. @ravikanth-nalla-hpe or @aasha-hpe  can provide examples.

Reply:
Example SRV records:

```
_sbps-hsn._tcp.<system domain>
_sbps-nmn._tcp.<system domain>
```

Reply:
> Might help to make match the IQN examples for the targets they'll see on a CSM system, based on worker name. @ravikanth-nalla-hpe or @aasha-hpe can provide examples.

Here are few IQN examples that we see on couple of target systems:
**Mug:**  iqn.2023-06.csm.iscsi:ncn-w002  <-------- ncn-w002 of Mug is configured with iSCSI
**Odin:**  iqn.2023-06.csm.iscsi:ncn-w005  <-------- ncn-w005 of Odin is configured with iSCSI
                 iqn.2023-06.csm.iscsi:ncn-w006 <-------- ncn-w006 of Odine is configured with iSCSI